recipes-siot/simpleiot/simpleiot/siot

```diff
@@ -13,8 +13,8 @@ mkdir -p $DATADIR
 case "$1" in
   start)
         echo -n "Starting $DESC"
-         check_dbs
-        start-stop-daemon -S -m -p $PIDFILE -b -x /usr/bin/siot
+        check_dbs
+        start-stop-daemon -S -m -p $PIDFILE -b -x /bin/sh -- -c 'SIOT_PORT=80 /usr/bin/siot'
```

Comment:
can we try
`start-stop-daemon -S -m -p $PIDFILE -b -x /usr/bin/env VAR1="Value"  'SIOT_PORT=80 /usr/bin/siot'`

Reply:
updated PR to use /usr/bin/env -- agreed, that is probably a little cleaner

Reply:
ok, I think I got it now